src/results-output/results-doc.ts

```diff
@@ -362,9 +358,8 @@ export function appendPrompt(onAppended?: OnAppendedCallback) {
     append(getPrompt(), onAppended);
 }
 
-export async function getFilePathForCurrentNameSpace(): Promise<[string, string]> {
+export async function getUriForCurrentNamespace(): Promise<vscode.Uri> {
     const ns = getNs();
     const info = await getSession().info(ns, ns);
-    const fileName = info.file;
-    return [fileName, vscode.Uri.parse(fileName).path];
+    return vscode.Uri.parse(info.file, true);
```

Comment:
It seems more reliable to pass around a `vscode.Uri` for cross platform reasons, and pick what you need from it when needed.